src/Drupal/DrupalExtension/ServiceContainer/DrupalExtension.php

```diff
@@ -199,6 +199,13 @@ public function configure(ArrayNodeDefinition $builder)
             end()->
           end()->
         end()->
+        // Screenshot path.
+        arrayNode('screenshot')->
+          children()->
+            scalarNode('path')->defaultValue('/tmp/screenshots')->end()->
+            scalarNode('failure_suffix')->defaultValue('F')->end()->
+          end()->
+        end()->
         end()->
         end();
```

Comment:
@kumikoono This should be:

```suggestion
        arrayNode('screenshot')->
          children()->
            scalarNode('path')->
              defaultValue('/tmp/screenshots')->end()->
              scalarNode('failure_suffix')->
                defaultValue('F')->end()->
              end()->
            end()->
          end()->
        end();
```

Reply:
Unchanged. The line is not too long.

Reply:
@kumikoono My point was to make it easier to find each pair of begin-end block by adding the indents for the nests.


```
children
  ...
->end()
```

```
arrayNode
  ...
->end()
```

```
ScalarNode
  ...
->end()
```

```
defaultValue(...)->end()  // No indent.
```